manager_dashboard/manager_dashboard/js/uploadProjects.js

```diff
@@ -21,14 +21,16 @@ function getFormInput() {
         ' (' + form_data.projectNumber + ')\n' +
         form_data.requestingOrganisation
 
-    // add teamId if visibility is not set to public
-    visibility = document.getElementById("visibility").value
-    if (visibility !== "public") {
-        form_data.teamId = visibility
-        maxTasksPerUser = document.getElementById("maxTasksPerUser").value
+    // add limit how many tasks a user can work on
+    maxTasksPerUser = document.getElementById("maxTasksPerUser").value
```

Comment:
It looks like you have a weird indentation on the next line.

Reply:
yep, fixed.